config/jobs/periodic/containerd/test-containerd-periodics.yaml

```diff
@@ -3,6 +3,10 @@ periodics:
     cluster: k8s-ppc64le-cluster
     decorate: true
     interval: 24h
+    reporter_config:
+        slack:
+          channel: 'prow-job-notifications'
+          report_template: 'Job {{.Spec.Job}} of type {{.Spec.Type}} ended with state {{.Status.State}}. <!subteam^S02N6DWBX0F> <{{.Status.URL}}|View logs>'
```

Comment:
The job states `failure` and `error` would be reported through the configuration at https://github.com/ppc64le-cloud/test-infra/blob/master/config/prow/config.yaml#L11:L13

Is this change intended to report even success states OR 
 only to change the alias name to `@prow-job-docker` in the notification message?

Reply:
I want to activate Slack notification in case of failure with this change. If I correctly understand the point, it is useless, because it is made by default by the file you linked. Change in reporter config is needed only if we want another name, or a notification on success.
So, except if I make another mistake, this PR is useless and can be closed. Can you confirm?